package.json

```diff
@@ -6,9 +6,15 @@
     "@testing-library/jest-dom": "^5.16.2",
     "@testing-library/react": "^12.1.3",
     "@testing-library/user-event": "^13.5.0",
+    "axios": "^0.26.0",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
+    "react-redux": "^7.2.6",
+    "react-router-dom": "^6.2.1",
     "react-scripts": "5.0.0",
+    "redux": "^4.1.2",
+    "redux-thunk": "^2.4.1",
+    "reselect": "^4.1.5",
     "web-vitals": "^2.1.4"
   },
```

Comment:
- [ ] OPTIONAL
Kindly ensure that your app is properly configured, so that is runs smoothly without breaking.

**Configuration error:**
![image](https://user-images.githubusercontent.com/12745474/155491526-32da523e-5c7c-47b3-9abd-49aae0220c5a.png)